app/assess/templates/components/application_overviews_table.html

```diff
@@ -102,6 +105,7 @@
                 <td class="govuk-table__cell"><a class="govuk-link" href="{{url_for('assess_bp.application',application_id=overview.application_id)}}">{{overview.project_name}}</a></td>
                 <td class="govuk-table__cell">{{asset_types[overview.asset_type]}}</td>
                 <td class="govuk-table__cell">&pound;{{ "{:,.2f}".format(overview.funding_amount_requested|int|round) }}</td>
+                <td class="govuk-table__cell">{{ overview.location_json_blob.country }}</td>
```

Comment:
```suggestion
                <td class="govuk-table__cell">{{ overview.location_json_blob.country if overview.location_json_blob else 'N/A'}}</td>
```

Could change `N/A` to a blank string or something else

Reply:
Change made as per suggestion 👍